.github/workflows/main-ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python package
```

Comment:
It's best to rename the workflow to «Tests» and the file to run-tests.yml, so that we understand what it does